pageserver/src/http/routes.rs

```diff
@@ -448,7 +448,7 @@ async fn timeline_create_handler(
                     .map_err(ApiError::InternalServerError)?;
                 json_response(StatusCode::CREATED, timeline_info)
             }
-            Err(tenant::CreateTimelineError::AlreadyExists) => {
+            Err(tenant::CreateTimelineError::Conflict) | Err(tenant::CreateTimelineError::AlreadyCreating)=> {
```

Comment:
Has rustfmt really given up on us:


```suggestion
            Err(tenant::CreateTimelineError::Conflict) | Err(tenant::CreateTimelineError::AlreadyCreating) => {
```

Could also:

```suggestion
            Err(tenant::CreateTimelineError::Conflict | tenant::CreateTimelineError::AlreadyCreating) => {
```

Reply:
applied in 611618b59